config/locales/el.yml

```diff
@@ -0,0 +1,341 @@
+---
+el:
```

Comment:
Yes that's true - not done yet, but I thought it needs to be on the folder for the locale to work - the translation is done on 85% and I wanted to try it on an instance before completing it!